<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titulo_historial_asistencias">
       <property name="styleSheet">
        <string notr="true">QLabel#titulo_historial_asistencias {
    font: 600 20pt &quot;Segoe UI&quot;;
    color: #4154F1;   /* Azul igual al figma */
}</string>
       </property>
       <property name="text">
        <string>Historial Asistencia </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="seleccione_curso">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 1px solid #d0d7e2;
    border-radius: 8px;
    padding: 6px 10px;
    background-color: #ffffff;
    color: #012970;
    font-size: 14px;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox:hover {
    background-color: #F6F9FF;
}</string>
         </property>
         <property name="placeholderText">
          <string>seleccione curso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nombre_estudiante">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #d0d7e2;
    border-radius: 8px;
    padding: 6px 10px;
    background-color: #ffffff;
    color: #012970;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #012970;
    background-color: #F6F9FF;
}
</string>
         </property>
         <property name="placeholderText">
          <string>nombre_estudiante</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buscar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    background-color: #4154F1;  /* Azul Figma */
    color: white;
    font: 10pt &quot;Segoe UI&quot;;
    padding: 8px 20px;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #2e3bd1;  /* Azul m치s oscuro al hover */
}</string>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fecha_asistencia">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QDateEdit {
    border: 1px solid #d0d7e2;
    border-radius: 8px;
    padding: 6px 10px;
    background-color: #ffffff;
    color: #012970;
    font-size: 14px;
}

/* Bot칩n desplegable del calendario */
QDateEdit::drop-down {
    border: none;
    width: 25px;
    subcontrol-origin: padding;
    subcontrol-position: center right;
}

/* Flecha personalizada */
QDateEdit::down-arrow {
    image: url(:/icons/icono_de_flecha.png);   /*Ruta correcta */
    width: 14px;
    height: 14px;
}

/* Hover igual que los dem치s */
QDateEdit:hover {
    background-color: #F6F9FF;
}
</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">QTableView {
    border: 1px solid #d0d7e2;
    border-radius: 8px;
    gridline-color: #d0d7e2;
    font-size: 14px;
    background-color: #ffffff;
    color: #012970;
    selection-background-color: #F6F9FF;
    selection-color: #012970;
}

QHeaderView::section {
    background-color: #F6F9FF;
    color: #012970;
    font-weight: bold;
    padding: 6px;
    border: none;
    border-bottom: 1px solid #d0d7e2;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="exportar_asistencia">
         <property name="styleSheet">
          <string notr="true">QPushButton#exportar_asistencia {
    border: none;
    background-color: #3BAE7E;   /* Verde Figma */
    color: white;
    font: 10pt &quot;Segoe UI&quot;;
    padding: 8px 20px;
    border-radius: 8px;
}

QPushButton#exportar_asistencia:hover {
    background-color: #2d8f68;   /* Verde m치s oscuro al hover */
}</string>
         </property>
         <property name="text">
          <string>Exportar Asistencia</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
